### PR TITLE
extract url from internal page query param

### PR DESCRIPTION
**Pull request by chung-lai**
See #3779, this should address the toggle issue not allowing the correct url.

https://user-images.githubusercontent.com/87332578/172492799-b901ac76-de62-401e-b153-bf27492bd0ea.mov

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

File: Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuView.swift [hunk 1]
```diff
@@ -36,7 +36,15 @@ class TrackingStatsViewModel: ObservableObject {
                 ]
             )
 
-            guard let domain = selectedTabURL?.host, Defaults[.cookieCutterEnabled] else {
+            var url = selectedTabURL
+            // extract url query param for internal pageg
+            if url != nil && url?.scheme == "internal" {
```
Comment (evancrow) [inline Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuView.swift:41]: Doesn't the `InternalURL` type have a method to unwrap?

Comment (chung-lai) [reply Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuView.swift:41]: Awesome! Will change

File: Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuView.swift [hunk 2]
```diff
@@ -36,7 +36,15 @@ class TrackingStatsViewModel: ObservableObject {
                 ]
             )
 
-            guard let domain = selectedTabURL?.host, Defaults[.cookieCutterEnabled] else {
+            var url = selectedTabURL
+            // extract url query param for internal pageg
```
Comment (evancrow) [inline Client/Frontend/Overlay/Content/Menu/TrackingProtection/TrackingMenuView.swift:40]: Nit: pages?

**Review by evancrow — APPROVED** [review]
LGTM - one nit

**Pull request merged**